### ISSUE TITLE
Write a Cayley driver for the current storage.go interface.

### ISSUE THREAD

**Issue by xllora** [google/badwolf#9]
As a first step towards unifying efforts with http://github.com/google/cayley we are going to target creating a driver implementation against Cayley.

**Comment by danbri** [comment]
Any news on this? (and #10) --- is interop with Cayley still under active investigation?

**Comment by xllora** [comment]
Yes, it is still on the cards. @barakmich and I had a quick conversation about it six month ago. Main issue, find some bandwidth to write a first initial driver. Likely it will show up over at https://github.com/xllora/bwdrivers, but given the other things we need to get right on BadWolf, it may still take a while.
